src/design-system.html

```diff
@@ -7,10 +7,55 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+   <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet"> 
     <link rel="stylesheet" href="styles/index.css" />
     <title>Design System</title>
   </head>
   <body>
-    <h1>Design System</h1>
+    <main class="page">
+      <h1>Design System</h1>
+      <h2>Buttons</h2>
+      <h3>Primary</h3>
+      <div class="row">
+        <button class="btn btn--primary">PRIMARY</button>
+        <button class="btn btn--primary" disabled="true">PRIMARY DISABLED</button>
```

Comment:
Attributos booleanos no HTML tu pode passar só nome do atributo que funciona tambem, acho que fica mais legivel.

```suggestion
        <button class="btn btn--primary" disabled>PRIMARY DISABLED</button>
```